cpp/extract_normdata_translations.cc

```diff
@@ -0,0 +1,225 @@
+
+/* We would like to have a tool that extracts translations for given
+   authority data
+   The german term is found in Field 150 
+   Currently there are two different kinds of translations:
+   TxTheo-Translations with the following definitions:
+
+   710: Körperschaft - fremdsprachige Äquivalenz        
+   711: Konferenz - fremdsprachige Äquivalenz  
+   700: Person - fremdsprachige Äquivalenz     
+   730: Titel - fremdsprachige Äquivalenz      
+   750: Sachbegriff - fremdsprachige Äquivalenz  
+   751: Geografikum - fremdsprachige Äquivalenz    
+
+   LoC/Rameau Translations:
+   
+   700: Person - Bevorzugter Name in einem anderen Datenbestand
+   710: Körperschaft - Bevorzugter Name in einem anderen Datenbestand              
+   711: Konferenz - Bevorzugte Benennung in einem anderen Datenbestand
+   730: Einheitstitel - Bevorzugter Name in einem anderen Datenbestand
+   750: Sachbegriff - Bevorzugte Benennung in einem anderen Datenbestand
+   751: Geografikum - Bevorzugter Name in einem anderen Datenbestand  
+   
+*/ 
+
+
+#include <iostream>
+#include <vector>
+#include <cstdlib>
+#include "Compiler.h"
+#include "MarcUtil.h"
+#include "MediaTypeUtil.h"
+#include "StringUtil.h"
+#include "Subfields.h"
+#include "util.h"
+
+// Languages to handle
+//enum languages { "en","fr" };
+
+const unsigned int NUMBER_OF_LANGUAGES = 2;
+const std::string languages_to_create_str = "en, fr";
+enum languages { en, fr };
+
```

Comment:
Please use an "enum class".  This can then replace the code for the last 3 lines.